mainchain-follower/mock/src/native_token.rs

```diff
@@ -0,0 +1,27 @@
+use async_trait::async_trait;
+pub use main_chain_follower_api::block::*;
```

Comment:
Why does it have to be public?

Reply:
it doesn't even need to be here. removed